src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/OpenApiHttpTriggerContext.cs

```diff
@@ -40,8 +40,9 @@ public class OpenApiHttpTriggerContext : IOpenApiHttpTriggerContext
         /// <summary>
         /// Initializes a new instance of the <see cref="OpenApiHttpTriggerContext"/> class.
         /// </summary>
-        public OpenApiHttpTriggerContext()
+        public OpenApiHttpTriggerContext(IOpenApiConfigurationOptions configOptions)
```

Comment:
Can we make this parameter as optional &ndash; `IOpenApiConfigurationOptions configOptions = null`?